core/frontend/src/types/wifi.d.ts

```diff
@@ -0,0 +1,43 @@
+export interface Network {
+    ssid: string
+    signal: number
+    locked: boolean
+    saved: boolean
+}
+
+export interface WifiStatus {
+    bssid: string
+    freq: int
+    ssid: string
+    id: int
+    mode: string
+    wifi_generation: int
+    pairwise_cipher: string
+    group_cipher: string
+    key_mgmt: string
+    wpa_state: string
+    ip_address: string
+    p2p_device_address: string
+    address: string
+    uuid: string
+}
```

Comment:
number instead of int

Reply:
Wait, `int` is not a valid type, how the tests are passing ??

Reply:
It's strange, indeed. Any idea?

Reply:
we should add a test to test the tests :eyes:

Reply:
This a must thing to fix and do it asap, if the typescript checks are not working, something is really wrong with the project.

Reply:
Apparently what's happening is that the 'int' was casted to 'unknown', thus accepting literally anything.
[There's a rule](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-explicit-any.md) for setting that.I will take a better look to see if it's the responsible for this behavior.